README.md

```diff
@@ -1,7 +1,7 @@
-# codewind-odo-extension
-Extension to Codewind providing support for odo projects: https://codewind.dev
+# Codewind ODO extension
+Extension to Codewind providing support for ODO projects: https://codewind.dev
 
-Add additional rules to support ODO extension
+## Add additional rules to support Codewind ODO extension
 ODO extension needs add additional rules for accessing OpenShift resources
```

Comment:
The ODO extension needs to add additional rules for accessing OpenShift resources.